### PR TITLE
edits for new SMP relationship

### PR DESCRIPTION
**Pull request by mikedurand**
added a few edits that put the SMP comparison in context, and provide a better jumping off point for a microstructure-related project.

### CODE REVIEW

**Comment by jomey** [comment]
Some typos are present .. working with Mike on Slack if address those later

**Review by jomey — APPROVED** [review]

**Comment by scottyhq** [comment]
@mikedurand could you please change this line form 'auto' to 'cache' https://github.com/snowex-hackweek/website/blob/6e5030e87e18192432a06f845c33cbcdb09b126b/book/_config.yml#L31

**Comment by jomey** [comment]
> @mikedurand could you please change this line form 'auto' to 'cache'
> 
> https://github.com/snowex-hackweek/website/blob/6e5030e87e18192432a06f845c33cbcdb09b126b/book/_config.yml#L31

Should we make this a separate PR since it is unrelated to tutorial changes?

**Comment by scottyhq** [comment]
> Should we make this a separate PR since it is unrelated to tutorial changes? @scottyhq 

yeah, probably best. but let's just see if it works here

**Comment by mikedurand** [comment]
sorry I didn't do a separate PR. I already pushed. please let me know how to fix if fix is needed.

File: book/references.bib [hunk 1]
```diff
@@ -198,4 +198,15 @@ @article{Leppanen2018
 pages = {404}, 
 number = {11}, 
 volume = {8}
+}
+
+@article{Calonne202,
```
Comment (scottyhq) [inline book/references.bib:203]: @mikedurand this needs to be "Calonne2020"

Comment (mikedurand) [reply book/references.bib]: argh sorry Scott! Fixed and pushed.

**Pull request merged**